cmd/convox/rack.go

```diff
@@ -69,6 +70,10 @@ func init() {
 						Name:  "stats",
 						Usage: "display process cpu/memory stats",
 					},
+					cli.BoolFlag{
+						Name:  "a, all",
```

Comment:
Might not be a big deal, but `-a` is used throughout the cli for `--app`. Perhaps to avoid confusion (even though this is used in the rack subcommand), just stick to `--all`?